src/modules/Submissions/components/List/ModalСreateSubmission/container.js

```diff
@@ -83,6 +85,16 @@ class ModalCreateSubmissionBuilder extends ContainerBuilder {
             options = files.map(({ name }) => name);
           } else {
             options = await getFileNamesFromZip(files[0]);
+            const fileName = files[0];
+            const parts = fileName.originalName.split("-");
+            let name = "";
+            let type = "";
+            if (parts.length >= 2) {
+              type = getTypeByShortPrefix(parts[0]);
+              name = parts[1];
+            } 
+            dispatchProps.setAnalysisType(type);
```

Comment:
If the split fails to deliver 2 pieces, we set both to parts to "". That doesn't make much sense to me.
If we have a hard requirement to have type, fail fast.
If not - at least preserve the originalName in the 'name'

Reply:
It is not a hard requirement.
Done.